src/content/docs/ai-monitoring/configure-ai-monitoring.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: 'Configure AI monitoring'
+metaDescription: 'You can apply certain configurations to your APM agents to change how your AI data appears in New Relic.'
+freshnessValidatedDate: never
+---
+
+Once you install AI monitoring, you can configure the default behavior of the agent or update your app to collect different kinds of data. 
+
+## Configure the agent [#configure-agents]
+
+Update default agent behavior for AI monitoring at these agent configuration docs: 
+
+* [Go](/docs/apm/agents/go-agent/configuration/go-agent-configuration/#ai-monitoring)
+* [Node.js](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#ai-monitoring)
+* [Python](/docs/apm/agents/python-agent/configuration/python-agent-configuration/#ai-monitoring)
+* [Ruby](/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration/#ai-monitoring)
+
+## Enable token count API [#enable-token]
+
+If you've disabled `ai_monitoring.record_content.enabled`, you may still want to know the number of tokens your app uses in an interaction. If you haven't disabled content from being recorded by the agent, you don't need to invoke the token count API. 
+
+Update your app code with the below code snippets: 
```

Comment:
I would recommend that we put a few notes here.

- New Relic does not recommend/endorse any specific LLM tokenizer for each model. It is the responsibility of the customer to pick an appropriate tokenizer for their supported models and languages.
- The examples are provided to illustrate how to use the LLM token count callback and are provided without warranty/support. It is the responsibility of the customer to implement the callback method appropriately for their use case.

I would also suggest that we do not state `Update your app code with the below code snippets`. Instead, we should state that these code snippets are reference guides only.